regen-network/regen-web
# feat: add social buttons on confirmations modals

## Description

Closes: regen-network/regen-registry#1492

- add a [ShareSection](https://github.com/regen-network/regen-web/pull/1772/files#diff-ef1ac4dc6afba71474bd66daf2fd7be2fd51c8d5e770241858d01d62f003c492) component
- use it on buy and retire confirmation modals

![image](https://user-images.githubusercontent.com/530644/216057811-f2e76a10-5f25-4ef7-9666-a4fdc49bde75.png)

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


#### Setting up backport PRs

After merging your PR to `dev`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR:

- If PR is a new feature: `@Mergifyio backport redwood hambach`.
- If PR is a bug fix: `@Mergifyio backport redwood hambach master`.

2. If your branch does have merge commits:

   a. Pull latest `dev`, `hambach` and `redwood` branches

   b. Create new branches for backports and merge `dev` (replace `<PR#>` with your PR #)

   ```
   git checkout -b hambach-backport-<PR#> hambach
   git merge dev
   git push origin hambach-backport-<PR#>
   git checkout -b redwood-backport-<PR#> redwood
   git merge dev
   git push origin redwood-backport-<PR#>`
   ```

   c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.

   d. If PR is a bug fix repeat the same process for `master` branch.

### How to test

1. https://deploy-preview-1772--regen-registry.netlify.app/storefront
2. test buy actions (from storefront or project card) and retire action (from portfolio)
3. check social buttons at the bottom of the confirmation modal

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
